woocommerce/woocommerce-ios
# DRY pods definitions in `Podfile`

### Description
The idea is to avoid version _definitions_ going out of sync. I don't think the resolved versions used in the different targets could get out of sync because of how CocoaPods works. Still, this seems like a worthy tidying to apply.

For context, I've been looking at the `Podfile` in preparation to update the internal pods rules to be ready to adopt SemVer coherently.

### Testing instructions
If CI is green, this change didn't break anything. Also notice that only the checksum changed in the `Podfile.lock`, further showing the `Podfile` changes didn't impact the resolved versions.

---
